packages/h5p-server/src/H5PEditor.ts

```diff
@@ -97,7 +100,8 @@ export default class H5PEditor {
             'copyright-semantics': defaultCopyrightSemanticsLanguageFile
         }).t,
         private urlGenerator: IUrlGenerator = new UrlGenerator(config),
-        private options?: IH5PEditorOptions
+        private options?: IH5PEditorOptions,
+        public contentUserDataStorage?: IContentUserDataStorage
```

Comment:
The options object should always be the last parameter. I had to add the ILockProvider parameter in the other PR and I put it into the objects. I think it would be best to refactor the constructor to have parameters like this storage: {libraryStorage: ILibraryStorage, contentStorage: IContentStorage, ...}. That way we avoid too many breaking changes when adding new optional parameters.

Reply:
Yeah. I put it last to avoid a breaking change since we basically only introduce a feature. I thought about the storage object as well and think that would be the best way to go.

Reply:
But I would do that in another PR as it is not really related to this feature. We shouldn't do too many things in one PR.